src/aviary/utils.py

```diff
@@ -73,7 +73,9 @@ def encode_image_to_base64(img: "np.ndarray") -> str:
 def check_if_valid_base64(image: str) -> str:
```

Comment:
With this name, I'd expect this function to return a boolean. 
Can you change the name to something like `validate_base64`?
Also, can you make sure the docstring says it returns the image in case it is a valid base64?